package.json

```
@@ -261,7 +261,7 @@
   },
   "engines": {
     "npm": "please-use-yarn",
-    "node": "14",
+    "node": "18.x",
```

nit: functionally equivalent to `18`, can we just use `18` to simplify?

```suggestion
    "node": "18",
```

---

Vercel specifically uses `18.x` notation which is why I stuck with it https://vercel.com/docs/concepts/functions/serverless-functions/runtimes/node-js